File: ai-service/app/routes/health_routes.py
from fastapi import APIRouter
from datetime import datetime, timezone
from app.core.config import settings

router = APIRouter()


@router.get("/health")
def health_check():
    return {
        "success": True,
        "message": "JobNuro AI service is running",
        "data": {
            "service": settings.app_name,
            "status": "healthy",
            "timestamp": datetime.now(timezone.utc).isoformat(),
        },
    }

File: backend/src/routes/health.routes.js
import express from "express";
import { sendSuccess } from "../utils/apiResponse.js";

const router = express.Router();

router.get("/", (req, res) => {
  return sendSuccess(res, {
    message: "JobNuro backend is running",
    data: {
      service: "JobNuro API",
      status: "healthy",
      timestamp: new Date().toISOString(),
    },
  });
});

export default router;